api/test/wfl/tools/workloads.clj

```diff
@@ -93,7 +93,7 @@
    :sample_lsid                     "broadinstitute.org:bsp.dev.sample:NOTREAL.03C17319",
    :analysis_version_number         1,
    :call_rate_threshold             0.98,
-   :genotype_concordance_threshold  0.98,
+   :genotype_concordance_threshold  0.95,
```

Comment:
(For my own understanding) this `arrays-sample` is just got give an example of what a real sample would look like for use in tests, and the threshold change is encompassed in the bump to WDL version.

Reply:
Yes. As stated in the ticket, this isn't strictly necessary, but without it someone will have to decide whether the discrepancy matters. I kinda did this to preclude questions like yours. ;-)